cgp/cartesian_graph.py

```diff
@@ -412,7 +412,10 @@ def to_sympy(self, simplify: Optional[bool] = True) -> List["sympy_expr.Expr"]:
 
         if not simplify:
             return sympy_exprs
-        else:  # simplify expression if desired
+        else:  # simplify expression if desired and possible
             for i, expr in enumerate(sympy_exprs):
-                sympy_exprs[i] = expr.simplify()
+                try:
+                    sympy_exprs[i] = expr.simplify()
+                except TypeError:
+                    Warning(f"Could not simplify expression: {expr}")
```

Comment:
```suggestion
                    RuntimeWarning(f"SymPy could not simplify the expression: {expr}")
```
like with exceptions, please be more specific

Reply:
~and shouldn't you nevertheless set `sympy_exprs[i] = expr`?~ nope ;)